products/messaging/frontend/Campaigns/hogflows/panel/HogFlowEditorPanelBuild.tsx

```diff
@@ -15,13 +15,26 @@ export const ACTION_NODES_TO_SHOW: CreateActionType[] = [
         type: 'function_email',
         name: 'Email',
         description: 'Send an email to the user.',
-        config: { template_id: 'template-email', inputs: {} },
+        config: {
+            template_id: 'template-email',
+            inputs: {
+                email: {
+                    templating: 'liquid',
+                },
+            },
+        },
     },
     {
         type: 'function_sms',
         name: 'SMS',
         description: 'Send an SMS to the user.',
-        config: { template_id: 'template-twilio', inputs: {} },
+        config: {
+            template_id: 'template-twilio',
+            inputs: {
+                to_number: { templating: 'liquid' },
+                message: { templating: 'liquid' },
```

Comment:
honestly every time you modify inputs here my alarm bells go off. I dont think we should be doing this tbh. Rather this should be part of the function itself. Otherwise we have two sources of truth and if we modify the template we wont realise what weve done